vault/resource_approle_auth_backend_role_secret_id.go

```diff
@@ -79,6 +79,13 @@ func approleAuthBackendRoleSecretIDResource(name string) *schema.Resource {
 				},
 			},
 
+			"reuse_wrapping_token": {
```

Comment:
I feel like `reuse_wrapping_token` might be a bit confusing for an operator, in that it somehow might relate to the way that the provider will be accessing vault,  but what it seems to be is a directive for specifying which accessor is provided in `wrapping_accessor`. Maybe we could rename it to `with_wrapped_accessor`

An alternative approach would be to rename it to `wrapping_accessor_type` with `schema.TypeString`, that takes either `default`, or `wrapped` as possible values?

Reply:
reuse_wrapping_token was originally suggested by @jasonodonnell  ( https://github.com/hashicorp/terraform-provider-vault/issues/976#issuecomment-862493014 ). I've got no particular attachment to it, so I'll change it to with_wrapped_accessor as suggested. 

Personally, I prefer boolean over a 2 possible value string, but if you really want it to be the second option you suggested, let me know and I'll see if I can make that work.

Reply:
Seems good!